### PR TITLE
Add licenses of  jquery-layout , cc #5500

### PR DESCRIPTION
**Pull request by yangwanting**
https://github.com/allpro/layout/blob/master/LICENSE.txt
@x09326

### CODE REVIEW

File: ajax/libs/jquery-layout/package.json [hunk 1]
```diff
@@ -18,6 +18,10 @@
       "*.css"
     ]
   },
+  "licenses": [
+    "MIT",
+    "GPL-3.0"
+  ]
```
Comment (x09326) [inline ajax/libs/jquery-layout/package.json]: @yangwanting 
lack a `,`, please add it.

**Comment by x09326** [comment]
LGTM @cdnjs/intern2 please help me review it, thanks.

**Comment by pvnr0082t** [comment]
@yangwanting You should say something like "Please help me review this pull request, thank you" after you tag your mentor or no one knows what you want to do.

**Comment by pvnr0082t** [comment]
LGTM

**Comment by x09326** [comment]
Close due to redo

**Pull request closed without merging**